catalog/app/containers/NavBar/NavBar.tsx

```diff
@@ -35,6 +35,8 @@ interface ItemProps extends M.MenuItemProps {
   href?: string
 }
 
+// FIXME: doesn't compile with Ref<unknown>
```

Comment:
Maybe, it's a place for a generic type. Something like this:

```ts
type ItemProps<T> = {
  component: T,
}
const Item<T> = forwardRef(props: ItemProps<T>, ref: Ref<T>) {
}
```

Just an idea for refactoring. I doubt, that it would be so simple :)

Reply:
And we can set default parameter `const Item<T = HTMLLIElement> `

Reply:
some googling gives me [that](https://wiki.c2.com/?FixmeComment#:~:text='FIXME'%20is%20for%20things%20which,and%20that%20are%20arguably%20broken.):

> 'FIXME' is for things which are definitely broken, but where you want to not worry about it for the moment. 'TODO' is for useful features, optimizations or refactorings that might be worth doing in the future.

so i guess it's ok

Reply:
> Just an idea for refactoring. I doubt, that it would be so simple

yeah, this component seems a little tricky bc the underlying component is based on the props, so we need to use some type assertion magic here, maybe smth w/ conditional types, idk